static/bidder-info/imds.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "http://{{.Host}}.technoratimedia.com/openrtb/bids/{{.Host}}"
+maintainer:
+  email: "eng-demand@imds.tv"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  # imds supports user syncing, but requires configuration by the host. contact this
+  # bidder directly at the email address in this file to ask about enabling user sync.
```

Comment:
There is a default iframe user sync defined, but then you've added a comment that configuration is required. Should you remove the default endpoint in that case? I wouldn't expect a host company to read this message if user syncing works out of the box.